engines/bladerunner/script/ai/moraji.cpp

```diff
@@ -81,6 +84,8 @@ void AIScriptMoraji::TimerExpired(int timer) {
 void AIScriptMoraji::CompletedMovementTrack() {
 	if (Actor_Query_Goal_Number(kActorMoraji) == kGoalMorajiRunOut) {
 		AI_Countdown_Timer_Reset(kActorMoraji, kActorTimerAIScriptCustomTask2);
+		if (_vm->_cutContent && Query_Difficulty_Level() == kGameDifficultyEasy)
+			Delay(1100); // Allow more time for the player to get away once Moraji is freed before the bomb goes off
```

Comment:
To be honest, I don't like this delay here. This introduces extra delay after Moraji has reached his "98" waypoint. Since in general Moraji's behavior is complicated in that scene and needs some thorough debugging (eg he never reaches his "97" waypoint), even though currently "it just works", I would prefer it if we avoided this altogether.